pom.xml

```diff
@@ -8,11 +8,6 @@
     <name>eBlocker - Common Project Definitions</name>
 
     <repositories>
-        <repository>
-            <id>maven.eblocker.org</id>
-            <name>eBlocker</name>
-            <url>https://maven.eblocker.org/m2/</url>
-        </repository>
```

Comment:
It seems to me that this is no longer necessary. Right?

Reply:
Unfortunately, this ist still necessary.

We had to mirror some packages from http://4thline.org/m2/, because this repo does not provide an HTTPS connection.

Reply:
Oh sorry. I have removed the commit from the PR.